src/main/java/org/assertj/core/api/ObjectEnumerableAssert.java

```diff
@@ -1311,11 +1311,49 @@
    * @return {@code this} to chain assertions.
    * @throws NullPointerException if given requirements is null.
    * @throws AssertionError if any element does not satisfy the requirements at the same index
-   * @throws AssertionError if there is not as many requirements as there are iterable elements.
+   * @throws AssertionError if there are not as many requirements as there are iterable elements.
    * @since 3.19.0
    */
   SELF satisfiesExactly(@SuppressWarnings("unchecked") Consumer<? super ELEMENT>... allRequirements);
 
+  /**
+   * Verifies that the actual iterable has the same size as the given consumers and 
+   * that at least one permutation of elements in the iterable exists that satisfies 
+   * the individual consumers in order.
+   * <p>
+   * Examples:
+   * <pre><code class='java'>
+   * List&lt;String&gt; starWarsCharacterNames = list("Luke", "Leia", "Yoda");
```

Comment:
join line and one space between them

Reply:
Thanks, I will align the line breaks in documentation example based on existing methods.